src/html/HTMLWriter.jl

```diff
@@ -512,11 +522,16 @@ struct HTML <: Documenter.Writer
         elseif size_threshold_warn > size_threshold
             throw(ArgumentError("size_threshold_warn ($size_threshold_warn) must be smaller than size_threshold ($size_threshold)"))
         end
+        if isnothing(example_size_threshold)
+            example_size_threshold = typemax(Int)
+        elseif example_size_threshold <= 0
```

Comment:
```suggestion
        elseif example_size_threshold < 0
```
Docstring suggests it can be set to `0`.